systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -141,6 +145,8 @@ public class Environment {
      */
     public static final String STRIMZI_POD_SET_RECONCILIATION_ONLY_ENV = "STRIMZI_POD_SET_RECONCILIATION_ONLY";
 
+    public static final String ST_FILE_SINK_PLUGIN_URL_ENV = "ST_FILE_SINK_PLUGIN_URL";
```

Comment:
I'm fine if you wanna keep the current naming both here as well as for the methods. But strictly speaking, you are adding both the File Sink and File Source connectors. So maybe the names can take that int account?

Reply:
Maybe I can use just `ST_FILE_PLUGIN_URL_ENV` as the package is named `connect-file` and also the `connect-file` plugin contains all addons related to file operations? In that case it will be also shorter than adding the `_AND_SOURCE` to the name.